test/types/streams.ts

```diff
@@ -0,0 +1,34 @@
+import asyncDone from "async-done";
+import { Readable, Stream } from "stream";
+
+function readableSuccess(): Readable {
+  return undefined as any;
```

Comment:
why are these returning undefined as any?

Reply:
These file are compiled but never run. I just needed to check that a function returning a `Readable` is accepted.

I did not care about the implementation of the task (runtime behavior is tested in `test/streams.js`). Using `any` allows to bypass type-checking locally ("trust me, this function returns a Readable"). 
I wrote the body of the functions for other kinds of objects because they are one-liners and it is more helpful to visualize a concrete example. In the case of streams, the boiler plate was slightly higher so I just used a dummy body.
I'll replace it with `return new Stream();` instead of replicating what's in the unit-tests.